src/lib/utils/idgen.js

```diff
@@ -0,0 +1,11 @@
+let id = 0;
```

Comment:
Nossa modal aceita um array de "actions", que serão um array de elementos.
Ex:
```javascript
const actions = [
      <Button>action1</Button>,
      <Button>action2</Button>,
      <Button>action3</Button>
    ];
```
Ao receber esse array na modal nós realizamos um .map para renderizar esses botões. O react orienta sempre a ter uma key para cada item do map ou for e o nosso lint não deixa a gente usar o index do item como key.

Por esses motivos eu adicionei o idgen.js, para gerar um id único nesses casos. Essa função foi copiada de outro lib de componentes react (react-materialize).